src/axom/primal/geometry/NURBSCurve.hpp

```diff
@@ -777,9 +783,12 @@ class NURBSCurve
     }
   }
 
-  /// \brief Normalize the parameterization of the knot vector
+  /// \brief Normalize the knot vector to the span of [0, 1]
   void normalize() { m_knotvec.normalize(); }
 
+  /// \brief Rescale the knot vector to the span of [a, b]
+  void rescale(T a, T b) { m_knotvec.rescale(a, b); }
```

Comment:
Does this handle the case where $a > b$ or $a == b$?
Please update the doxycomment